core/src/main/java/com/linecorp/armeria/common/util/AsyncLoader.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.util;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+/**
+ * An {@link AsyncLoader} to atomically load, cache and update value.
```

Comment:
I think it doesn't make much sense to refer to itself when defining something, especially in the first sentence of the definition. Can we rephrase this?

Reply:
✅ updated!